docs/en/docs/sql-manual/sql-reference/Data-Types/JSONB.md

```diff
@@ -35,8 +35,775 @@ under the License.
     2. JSONB format is more efficient. Using jsonb_extract functions on JSONB format is 2-4 times faster than get_json_xx on JSON STRING format.
 
 ### example
-    refer to jsob tutorial.
+A tutorial for JSONB datatype including create table, load data and query.
+
+### create database and table
```

Comment:
level 4

Reply:
And pay a attention to the following level

Reply:
thanks for remind, fixed